conftest.py

```diff
@@ -113,34 +125,47 @@ def driver(
     opt_headless: bool,
     opt_implicit_timeout: int,
     set_prefs: List[Tuple],
+    opt_ci: bool,
+    opt_window_size: str,
     env_prep,
 ):
     """
     Return the webdriver object.
 
     All arguments are fixtures being requested.
```

Comment:
i think it would be preferable to have sentences explaining what each of the arguments are for like in the rest of your files for consistency

Reply:
I'll add more docs